ui/app/components/currency-display/currency-display.container.js

```diff
@@ -16,6 +17,7 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   const {
     value,
     numberOfDecimals = 2,
+    nativeCurrency = ETH,
```

Comment:
We can pull this from state above (in `mapStateToProps`) and then we shouldn't need a default

Reply:
yes. as you already mentioned and logically we don't have to apply this fix at all.
but as you can see,  without this fix, currency-display do not show the correct value...

Reply:
drop last change